### DISCUSSION TITLE
1.0.0-alpha.4

### DISCUSSION THREAD

**Discussion by mxcl** [pkgxdev/pkgx#736] [Announcements]
## What's Changed
* `--which` -> `--provider`
* ensure shellcode can be called multiple times
* recommend `tea integrate` when running plain env commands
* handle action.yml without node by @mxcl in https://github.com/teaxyz/cli/pull/734

**Full Changelog**: https://github.com/teaxyz/cli/compare/v1.0.0-alpha.3...v1.0.0-alpha.4

<hr /><em>This discussion was created from the release <a href='https://github.com/teaxyz/cli/releases/tag/v1.0.0-alpha.4'>1.0.0-alpha.4</a>.</em>

**Comment by davdroman** [comment]
> --which -> --provider

Lovely 👌

**Comment by gaggle** [comment]
Loving these quick releases ❤️

 Is there awareness around the latest package didn't release to brew? Brew still reports `alpha.3`:

```sh
brew info teaxyz/pkgs/tea-cli
==> teaxyz/pkgs/tea-cli: stable 1.0.0-alpha.3 (bottled)
builders ❤️ tea
https://tea.xyz
Conflicts with:
  tea (because both install `tea` binaries)
/usr/local/Cellar/tea-cli/1.0.0-alpha.3 (2 files, 100.3MB) *
  Poured from bottle on 2023-09-15 at 19:21:59
From: https://github.com/teaxyz/homebrew-pkgs/blob/HEAD/tea-cli.rb
License: Apache-2.0
==> Dependencies
Build: deno ✘
==> Caveats
try it out:

    tea node@18 --eval 'console.log("brewing up some tea")'
```

The [CLI repo's brew pipeline](https://github.com/teaxyz/cli/actions/runs/6213817748) triggered [this homebrew-pkgs pipeline](https://github.com/teaxyz/homebrew-pkgs/actions/runs/6213818469/job/16865189676) which failed.

**Reply by jhheider** [reply]
yeah, the alphas require a manual run. should be updated now: https://github.com/teaxyz/homebrew-pkgs/actions/runs/6300594356/job/17103782567
